### DISCUSSION TITLE
Friends :)

### DISCUSSION THREAD

**Discussion by TheSKid0110** [BullyHunter32/methamphetamine-public#3] [General]
I cant add you as a friend on discord. You used to call me skid. I somehow found you. Fix friend :1st_place_medal:

**Comment by BullyHunter32** [comment]
You what?

**Reply by TheSKid0110** [reply]
Friends :)

**Comment by TheSKid0110** [comment]
I wasnt joking in the part where i said i cant add you as a friend pls fix

**Reply by BullyHunter32** [reply]
I will in ~3 weeks after my final exams are over and done with. After that I may add No-Recoil and spinbot
